api/feature_flags.go

```diff
@@ -0,0 +1,25 @@
+package api
+
+import (
+	"fmt"
+)
+
+type FeatureFlagsService struct {
+	client *Client
+}
+
+type FeatureFlag string
+
+type FeatureFlags struct {
+	Flags []FeatureFlag `json:"flags,omitempty"`
+}
+
+type FeatureFlagsResponse struct {
+	Data FeatureFlags `json:"data"`
+}
+
+func (svc *FeatureFlagsService) GetFeatureFlagsMatchingPrefix(prefix string) (response FeatureFlagsResponse, err error) {
+	apiPath := fmt.Sprintf("%s/%s", apiV2FeatureFlags, prefix)
+	err = svc.client.RequestDecoder("GET", apiPath, nil, &response)
+	return response, err
```

Comment:
you're using named returns, the recommendation is to use naked returns:
```suggestion
	return
```